bin/tictactoe

```diff
@@ -1,3 +1,21 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
+
```

Comment:
For lines 5-21. Lets add a new cli.rb file in the lib directory and turn this into a class with callable instance methods.

Then we can use something like this on line 5

CLI.new.start_game